src/app/patient-dashboard/common/ovc-snapshot/ovc-snapshot.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="program">
+<div class="enrolled">
+  <div *ngIf="ovcEnrollment" class="program-header-wrapper">
+    <h2 class="program-header">OVC Program (Enrolled on
+      {{dateEnrolled}})
+    </h2>
+  </div>
+  <div *ngIf="!ovcEnrollment" class="program-header-wrapper">
+    <h2 *ngIf="!exitedCare" class="program-header">OVC Program (Not Enrolled)
+    </h2>
+    <h2 *ngIf="exitedCare" class="program-header">OVC Program (Completed Program)
```

Comment:
Would it be useful to indicate when they completed the program?

Reply:
Added